app/assets/v2/css/quests.css

```diff
@@ -1400,3 +1400,73 @@ body.green .tl{
   }
 }
 
+/* Responsive for iPhone 6/7/8/Plus & X */
+
+@media (max-width: 500px) {
+  body.quest_battle{
+    background-size: 400% !important;
+  }
+  body.back6.quest_battle,
+  body.back7.quest_battle,
+  body.back8.quest_battle,
+  body.back9.quest_battle{
+    background-size: 250% !important;
+  }
+
+  /* Submit quest response button */
+
+  #cta_button a, a.button {
+    line-height: 150px;
+    padding: 10px;
+
+  body.stage_4 #cta_button {
+  margin-top: 150%;
+  }
+
+  .quest-card {
+    margin-left: 20% !important;
+  }
+  .back_gradient {
+    background-size: 100% 600px !important;
+  }
+  .demo {
+    max-width: 90%;
+    margin-left: 5%;
+  }
+
+  .tr {
+    left: 279px;
+    right: 70px;
+  }
+  .tl {
+    position: absolute;
+    top: -25px;
+    left: -25px;
+    }
+
+   #header {
+  font-size: 6px;
+  top: 35%;
+  }
+
+  body.stage_4 #header {
+    margin-top: -5px;
+   }
+
+  #desc {
+    height: 80px;
+    overflow-y: scroll;
+  }
+  body.stage_4 #desc {
+    margin-top: -2px;
+    }
+  #announce{
+    margin-top: 150px;
+  }
+  #announce img{
+    margin-bottom: 30px;
+  }
+  #leaderboard .inner_row {
+    height: 130px;
+  }
+}
```

Comment:
^ could we ensure there is an 
- empty line after every style declaration 
- `xyz{` -> `xyz {`